content/en/blog/_posts/2024-02-22-k8s-book-club/index.md

```diff
@@ -0,0 +1,126 @@
+builds a reliable community.
+
+Secondly, making the sessions interesting and interactive has been vital. For instance, I often
+introduce pop-up quizzes during the meetups, which not only tests members' understanding but also
+adds an element of fun. This approach keeps the content relatable and helps members understand how
+theoretical concepts are applied in real-world scenarios.
+
+## Topics covered in the Book Club
+
+**FSM**: The main topics of the books have been Kubernetes, GitOps, Security, SRE, and
+Observability: is this a reflection of the cloud native landscape, especially in terms of
+popularity?
+
+**CS**: Our journey began with 'Production Kubernetes', setting the tone for our focus on practical,
+production-ready solutions. Since then, we've delved into various aspects of the CNCF landscape,
+aligning our books with a different theme.  Each theme, whether it be Security, Observability, or
+Service Mesh, is chosen based on its relevance and demand within the community. For instance, in our
+recent themes on Kubernetes Certifications we brought the book authors into our fold as active
+hosts, enriching our discussions with their expertise.
+
+**FSM**: I know that the project had recent changes, namely being integrated into the CNCF as a
+[Cloud Native Community Group](https://community.cncf.io/). Could you talk a bit about this change?
+
+**CS**: The CNCF graciously accepted the book club as a Cloud Native Community Group. This is a
+significant development that has streamlined our operations and expanded our reach. This alignment
+has been instrumental in enhancing our administrative capabilities, similar to those used by
+Kubernetes Community Days (KCD) meetups. Now, we have a more robust structure for memberships, event
+scheduling, mailing lists, hosting web conferences, and recording sessions.
+
+**FSM**: How has your involvement with the CNCF impacted the growth and engagement of the Kubernetes
+Book Club over the past six months?
+
+
+**CS**: Since becoming part of the CNCF community six months ago, we've witnessed significant
+quantitative changes within the Kubernetes Book Club. Our membership has surged to over 600 members,
+and we've successfully organized and conducted more than 40 events during this period. What's even
+more promising is the consistent turnout, with an average of 30 attendees per event. This growth and
+engagement are clear indicators of the positive influence of our CNCF affiliation on the Kubernetes
+Book Club's reach and impact in the community.
+
+
+## Joining the Book Club
+
+FSM: For anyone wanting to join, what should they do?
+
+**CS**: There are three steps to join:
+
+- First, join the [Kubernetes Book Club
+  Community](https://community.cncf.io/kubernetes-virtual-book-club/)
+- Then RSVP to the
+  [events](https://community.cncf.io/kubernetes-virtual-book-club/#:~:text=Share-,Upcoming%20events,-Feb%208%2C%202024)
```

Comment:
```suggestion
  [events](https://community.cncf.io/kubernetes-virtual-book-club/)
```

That kind of fragment identifier is rather [loosely](https://wicg.github.io/scroll-to-text-fragment/) standardized; I tend to avoid it in documents that will live with us a long time.

Reply:
Thanks, I'll consider this if I encountered them again.